examples/mobile/UIComponents/components/controllers/checkbox-list-selected.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../lib/tau/mobile/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page" id="checkbox-list-selected-demo">
+		<header>
+			<div class="ui-appbar-left-icons-container">
+				<label>All
+					<input class="ui-li-area-b ui-checkbox-all" type="checkbox"/>
+				</label>
+			</div>
+			<div class="ui-appbar-title-container">
+				<span class="ui-appbar-title">3 selected</span>
+			</div>
+			<div class="ui-appbar-action-buttons-container">
+					<button class="ui-btn ui-btn-icon ui-btn-icon-search" data-style="flat"></button>
+					<button class="ui-btn ui-btn-icon ui-btn-icon-more" data-style="flat"></button>
+			</div>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview ui-content-area">
+				<li class="ui-li-flex">
```

Comment:
I think we don't need to 'ui-li-flex' anymore because 'li' element has the 'flex' style by default.
We can replace the code with follwing code.
```
<li>
  <div class="ui-li-text">
    <span class="ui-li-text-title">
      Selection item 01
    </span>
  </div>
  <div class="ui-li-action">
    <input type="checkbox" />
  </div>
</li>
```
But some new applied styles are not merged yet.
So I will change the code after merging this PR.

Reply:
Thank you for suggestion :) . This PR is in progress but I wonder that close this PR because this is rather implementation of multi selection list.

Reply:
@HunseopJeong 
I saw that you have plans to make this list.

Reply:
@HunseopJeong 
I don't have much information on the github about what you do (issues on github) because some elements are common like these checkbox on listview.

Reply:
@TomaszLukawskiSam,
Okay,, I also make the issue to share my work : )
Anyway I already said, you don't apply it in this PR.
I will apply it after merging this PR.
And check it my changes and follow it after your PRs.